ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -64,11 +68,14 @@ describe('AppsPanel', () => {
   const appsStub = new AppsApiStub();
   const runtimeStub = new RuntimeApiStub();
   beforeEach(() => {
-    serverConfigStore.set({ config: defaultServerConfig });
-    environment.showAppsPanel = true;
+    serverConfigStore.set({
+      config: { ...defaultServerConfig, enableCromwellGKEApp: true },
+    });
     registerApiClient(AppsApi, appsStub);
+    appsStub.createApp = () => Promise.resolve({});
```

Comment:
is this needed?  the stub exists (added recently)

Reply:
I did this out of fear of the issue described [here](https://github.com/all-of-us/workbench/pull/7450#discussion_r1122321488). Adding workaround code probably isn't the right path forward, though. The tests still pass when I remove it, so I'll remove it.